Cargo.toml

```diff
@@ -10,6 +10,5 @@ license = "MIT"
 keywords = ["iron", "web", "persistence", "sharing", "global"]
 
 [dependencies]
-iron = { version = "0.2", default-features = false }
```

Comment:
default-features = false is no longer needed. Same for all other PRs you opened.